fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenter.kt

```diff
@@ -86,6 +88,13 @@ class BrowserToolbarCFRPresenter(
         }
     }
 
+    private fun shouldShowCFR(): Boolean {
+        return settings.shouldShowTotalCookieProtectionCFR && (
+            !settings.shouldShowCookieBannerReEngagementDialog() ||
+                settings.openTabsCount >= CRF_MINIMUM_NUMBER_OPENED_TABS
+            )
+    }
```

Comment:
Thank you! 👍

Reply:
Thanks for proposing an alternative solution :)